mesonbuild/wrap/wrap.py

```diff
@@ -81,7 +81,7 @@ def open_wrapdburl(urlstring: str, allow_insecure: bool = False, have_opt: bool
             msg = f'WrapDB connection failed to {urlstring} with error {excp}.'
             if isinstance(excp.reason, ssl.SSLCertVerificationError):
                 if allow_insecure:
-                    mlog.warning(f'{msg}\n\n    Proceeding without authentication.')
+                    mlog.warning(f'{msg}\n\n    Proceeding without authentication.', fatal=False)
```

Comment:
This CLI subtool doesn't have a `--fatal-meson-warnings` flag.

Reply:
Oh, right. For some reason I thought this was a common path when fetching sources as well

Reply:
Hold on, that's a good point and we do. We auto-retrieve at least patch_url from the wrapdb when doing subproject fallback.

Reply:
And triple-checking, we cannot actually hit the SSL codepath in subproject fallback, because we don't support `--allow-insecure` except in the wrap subtool. Subproject fallback will always error out instead of emitting this `mlog.warning()`.